helpers/bot_base.py

```diff
@@ -246,23 +246,23 @@ async def on_error(self, event_method: str, *args: Any, **kwargs: Any) -> None:
 
         error_channel: discord.TextChannel = self.get_channel(880181130408636456)  # type: ignore # known ID
 
-        to_send = f"```yaml\nAn error occurred in an {event_method} event``````py" f"\n{traceback_string}\n```"
+        to_send = f"```yaml\nAn error occurred in an {event_method} event``` ```py" f"\n{traceback_string}\n```"
         if len(to_send) < 2000:
             try:
                 await error_channel.send(to_send)
 
             except (discord.Forbidden, discord.HTTPException):
 
                 await error_channel.send(
-                    f"```yaml\nAn error occurred in an {event_method} event``````py",
+                    f"```yaml\nAn error occurred in an {event_method} event```y",
```

Comment:
```suggestion
                    f"```yaml\nAn error occurred in an {event_method} event```",
```

Reply:
whoops